worker-default-configs/config/cfg~caf~worker~RabbitWorkerQueueConfiguration.js

```diff
@@ -19,5 +19,6 @@
             || (getenv("CAF_WORKER_BASE_QUEUE_NAME") || getenv("CAF_WORKER_NAME") || "worker") + "-in",
     retryQueue: getenv("CAF_WORKER_RETRY_QUEUE") || undefined,
     rejectedQueue: "worker-rejected",
-    retryLimit: getenv("CAF_WORKER_RETRY_LIMIT") || 10
+    retryLimit: getenv("CAF_WORKER_RETRY_LIMIT") || 10,
+    maxPriority: getenv("CAF_RABBITMQ_MAX_PRIORITY") || 255
```

Comment:
For backward compatibility it would be better if the default was `undefined`, so that the `worker-queue-rabbit` code could take care of defaulting to `255` if the parameter is not specified.